exercise/cache/points.py

```diff
@@ -318,6 +318,7 @@ def check_reveal_rule() -> None:
                     'url': submission.get_url('submission-plain'),
                     'feedback_revealed': True,
                     'feedback_reveal_time': None,
+                    'submission_hash' : submission.meta_data.get('submission_hash', None), #Duplicate Check: We insert the submission hash to Cache
```

Comment:
Now that we have moved the hash comparisons to the frontend JavaScript, we should think again if the hashes are even needed in the cache. Does the current code use the cache when rendering the hashes in the html template (exercise_plain.html, data attributes)?